ert_gui/tools/file/file_dialog.py

```diff
@@ -59,9 +60,31 @@ def _stop_thread(self):
         self._thread.quit()
         self._thread.wait()
 
+    def _get_file_line_count(self):
+        file_line_count = len(self._file.readlines())
+        self._file.seek(0)
+        return file_line_count
+
+    def _calculate_min_width_and_height(self):
```

Comment:
Is it possible to split this function into two functions? `_calculate_min_width` and `_calculate_min_height` or `_min_width` and `_min_height`. Just as a rule of thumb try avoiding returning more than one variable at a time if it can be avoided. It can easily become confusing, this is not one of the cases where things are confusing but just to keep things consistent.

Reply:
thanks for the pointer, that's a good rule to remember! will do.